documentation/docs/quick-start.md

```diff
@@ -37,7 +37,7 @@ A Cloud Agent can issue, hold, and verify [verifiable credentials (VCs)](/docs/c
 
 It is maintained as an open source lab through the [Hyperledger Foundation: Open Enterprise Agent](https://labs.hyperledger.org/labs/open-enterprise-agent.html).
 
-More in depth documentation about [Cloud Agent](/docs/concepts/glossary/#cloud-agent-aka-prism-cloud-agent) can be found [here](/docs/atala-prism/prism-cloud-agent/overview).
+More in depth documentation about [Cloud Agent](/docs/concepts/glossary/#cloud-agent) can be found [here](/docs/identus/cloud-agent/overview).
```

Comment:
The link /docs/identus/cloud-agent/overview does not exist. You may do in 2 steps:
- rename the text only 
- update the links

Reply:
fixed